tools/test-proxy/Azure.Sdk.Tools.TestProxy/RecordingHandler.cs

```diff
@@ -50,6 +50,8 @@ public RecordingHandler(string targetDirectory)
         private static readonly string[] s_contentRequestHeaders = new string[] {
```

Comment:
I think this was meant to be removed?

Reply:
Nevermind!

Reply:
Yeah, commit just wasn't included. Fixed now.